### ISSUE TITLE
Fix metadata display and navigation functions on blackboard nodes

### ISSUE THREAD

**Issue by mbrush** [NCATS-Tangerine/tkbio#62]
At present, tk.bio  behavior when clicking on a concept node in the blackboard is variable.  For some concepts (mostly Wikidata ones) a nice table showing concept metadata appears, and the user is given the option to  show relations' for that concept in the statement table.  But for many concepts (e.g. those from Biolink), no metadata is returned, and the 'show relations' button does not work. This is a significant barrier to navigation of the data in the way we would like to support.

Image below shows dialog box returned after clicking a concept node from Biolink - no metadata returned as for most Wikidata nodes, and also the 'show relations' box is inactive/greyed out.
  
![image](https://user-images.githubusercontent.com/5184212/30606948-4007f370-9d28-11e7-8eb3-c396cf7766c3.png)

**Comment by mbrush** [comment]
can we close this issue - given that the duplicated issue in the beacon-aggregator repo was closed?

**Comment by RichardBruskiewich** [comment]
No... sorry... issue #58 was actually a duplication of this issue, which is why it was closed. 

That said, I hope to have this fixed before the hackathon, if not sooner (perhaps today, Friday 20th October, as I write this comment?)
